packages/storybook/.storybook/preview-head.html

```diff
@@ -4,3 +4,6 @@
   delete Document.prototype.adoptedStyleSheets;
 </script>
 <link rel="stylesheet" id="cxl-font-css"  href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i,900|Libre+Baskerville:400,400i,700" type="text/css" media="all">
+
+<!-- Load JW Player -->
+<script src="https://cdn.jwplayer.com/libraries/ph5ocxbr.js"></script>
```

Comment:
NB: The library url will have to be signed in production. We're currently loading this up on PHP. If we can add some type of jwt/env to approximate url signing in storybook that would be good for testing. I can provide credentials.

Reply:
Ok, let's discuss this in more detail on slack